kernel/cClosure.ml

```diff
@@ -315,13 +315,13 @@ end = struct
     | _ -> assert false
 
   let[@inline] red_state x = match x land 0b1100 with
-    | 0b0000 -> Norm
+    | 0b0000 -> Ntrl
     | 0b0100 -> Cstr
     | 0b1000 -> Whnf
     | 0b1100 -> Red
     | _ -> assert false
 
-  let[@inline] neutr x = x lor 0b1000 (* Whnf|Norm -> Whnf | Red|Cstr -> Red *)
+  let[@inline] neutr x = x lor 0b1000 (* Whnf|Ntrl -> Whnf | Red|Cstr -> Red *)
```

Comment:
While we're at it this function's name may be confusing too (called `neutr` but removes `Ntrl`)

Reply:
Suggestion welcome.